### ISSUE TITLE
No Connection to 3cx pbx possible

### ISSUE THREAD

**Issue by mr2017x** [ictinnovations/ictfax#52]
I Installed ictfax as shown in manual and added freeswitch bug-fix as told in other issue (is running then fine), but 
1. as i see neither trunk nor
2. extensions are building up a connection/registration to the pbx and

3. all sent faxes are stuck at processing. 

how to deal with this?

Q: Do i need to set up trunk with user/password (if yes: extension no or auth id with password?
Q: How do i set one extension to one user?
Q: Furthermore why can i see more options on http://demo.ictfax.org/#/pages/did/did than on my installed ictfax implementation?

Thanks in advance!

**Comment by fizaict** [comment]
Yes, you need to set up the trunk so your faxes can be delivered.

Currently, there is no interface in the GUI to assign an extension to the user.

On demo.ictfax.org ICTFax SP Edition is installed which has more features than the Open Source version.

**Comment by plochu** [comment]
@mr2017x the easiest way is to register ICTFax as Extension in 3CX.
But as ICTFax Freeswitch is not configured well you need to remember about important thing on 3CX side. **Extension Number** and **Authentication ID** needs to be the same. This will deal with outgoing faxes.

Reciving is more complicated because ICTFax uses separate trunks to send and recieve faxes.

**Comment by mr2017x** [comment]
> Yes, you need to set up the trunk so your faxes can be delivered.
> 
> Currently, there is no interface in the GUI to assign an extension to the user.
> 
> On demo.ictfax.org ICTFax SP Edition is installed which has more features than the Open Source version.

ok ,set up, put in ext id = auth id + password, nothing happens, no registration issued as i can see on pbx' side.

is there a cli command i didn't see for that?

ok, so another code basis, where is the open source approach? and furthermore, is there a "hidden" installation source which is running smooth?

> @mr2017x the easiest way is to register ICTFax as Extension in 3CX.
> But as ICTFax Freeswitch is not configured well you need to remember about important thing on 3CX side. **Extension Number** and **Authentication ID** needs to be the same. This will deal with outgoing faxes.
> 
> Reciving is more complicated because ICTFax uses separate trunks to send and recieve faxes.

i guess so, i am in that business with pbx since several years didn't hazzle with it, also ext id, auth id (if differ) and so on is no problem, but here it seems - it's also no problem - as it doesn't issue a registration from ictfax towards pbx, but it also doesn't seem to log anything so far...?

did i miss some log or is this version just buggy (in reference to freeswitch error i had to fix and so on?)

furthermore is ictfax still developed?

**Comment by motherboard1999** [comment]
@plochu @mr2017x any updates?
I created in 3CX a new fax extension. Then I add a new SIP Trunk in ICTFax. The username and password is the same as in 3CX. As host I took the 3CX IP and as port I tried 5060. The other fields are empty. Now if I try to send a fax, it stuck at "processing". How you guys resolved this?

**Comment by fizaict** [comment]
Please share the Freeswitch log

**Comment by motherboard1999** [comment]
[out.log](https://github.com/ictinnovations/ictfax/files/7202725/out.log)
